nuklear.h

```diff
@@ -29,7 +29,7 @@
 /// This is a minimal state immediate mode graphical user interface toolkit
 /// written in ANSI C and licensed under public domain. It was designed as a simple
 /// embeddable user interface for application and does not have any dependencies,
-/// a default renderbackend or OS window and input handling but instead provides a very modular
+/// a default render backend or OS window and input handling but instead provides a very modular
```

Comment:
This change needs to be made in [src/HEADER](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/HEADER).

Reply:
I don't understand why these 2 commits were not merged. The comment needs correction and there is still the problem with the double include maths (not a huge issue but annoying).

Reply:
@crazyBaboon `nuklear.h` gets concatenated from source files in `src/` using `src/paq.sh` (or `src/paq.bat` on Windows), so the changes need to take place first in the files in `src/`.

Feel free to read the [reviewers guide](https://github.com/Immediate-Mode-UI/Nuklear#reviewers-guide ) to see what'll our reviewers usually look for :wink:.

Reply:
Hi @dumblob, thanks for pointing out the [reviewers guide](https://github.com/Immediate-Mode-UI/Nuklear#reviewers-guide). 
The changes of this PR mostly affect files in the `demo/*` folder, since they are just remove double includes in the demo files. Is there anything wrong with this?

Reply:
>Is there anything wrong with this?

As you wrote **mostly affect**, so my answer is *there is **mostly** nothing wrong* :wink:. For any change in `nuklear.h`, the change has to be made in `src/...` and then `nuklear.h` generated. If you wouldn't touch `nuklear.h` in this PR, we'd merge it immediately, but because it changes `nuklear.h`, we need to ensure the quality and consistency gets preserved.

Reply:
@dumblob Sorry I understand now!